src/uwtools/config/tools.py

```diff
@@ -90,7 +90,7 @@ def realize_config(
     """
     NB: This docstring is dynamically replaced: See realize_config.__doc__ definition below.
     """
-    input_obj, input_format = _realize_config_input_setup(input_config, input_format)
```

Comment:
The returned `input_format` value was never used, so is no longer returned by `_realize_config_input_setup()` (see below).